tests/contrib/aiobotocore/test.py

```diff
@@ -59,11 +61,32 @@ def test_s3_client(self):
         eq_(span.resource, 's3.listbuckets')
         eq_(span.name, 's3.command')
 
+    @mark_asyncio
+    def _test_s3_put(self):
+        params = dict(Key='foo', Bucket='mybucket', Body=b'bar')
+
+        with aiobotocore_client('s3', self.tracer) as s3:
+            yield from s3.create_bucket(Bucket='mybucket')
+            yield from s3.put_object(**params)
+
+        spans = [trace[0] for trace in self.tracer.writer.pop_traces()]
+        assert spans
+        self.assertEqual(len(spans), 2)
+        self.assertEqual(spans[0].get_tag('aws.operation'), 'CreateBucket')
+        self.assertEqual(spans[0].get_tag(http.STATUS_CODE), '200')
+        self.assertEqual(spans[0].service, 'aws.s3')
+        self.assertEqual(spans[0].resource, "s3.createbucket")
+        self.assertEqual(spans[1].get_tag('aws.operation'), 'PutObject')
+        self.assertEqual(spans[1].resource, "s3.putobject")
+        self.assertEqual(spans[1].get_tag('params.Key'), stringify(params['Key']))
+        self.assertEqual(spans[1].get_tag('params.Bucket'), stringify(params['Bucket']))
+        self.assertEqual(spans[1].get_tag('params.Body'), stringify(params['Body']))
+
     @mark_asyncio
     def test_s3_client_error(self):
         with aiobotocore_client('s3', self.tracer) as s3:
             with assert_raises(ClientError):
-                yield from s3.list_objects(Bucket='mybucket')
+                yield from s3.list_objects(Bucket='doesnotexist')
```

Comment:
ha, `moto` might be keeping a reference to the same list of S3 buckets from previous tests, probably need to reset `moto` in `tearDown` or something.

This change isn't necessary since you have a way around it, but just worth pointing out the potential cause here.